src/main/java/control/TimelineController.java

```diff
@@ -80,7 +69,10 @@ public void addCameraShot(int cameraIndex, String name, String description,
         log.info("Adding CameraShot to Timeline");
 
         CameraShot newShot = new CameraShot(name,description, startCount, endCount);
-        this.project.getCameraTimelines().get(cameraIndex).addShot(newShot);
+        this.controllerManager.getScriptingProject()
+                              .getCameraTimelines()
+                              .get(cameraIndex)
```

Comment:
I think this way it is structured better. It may take a few more lines but the method calls are very nicely aligned